pkg/dockerregistry/server/repositorymiddleware.go

```diff
@@ -101,9 +104,16 @@ func newRepositoryWithClient(registryClient client.Interface, quotaClient kclien
 		}
 	}
 
-	nameParts := strings.SplitN(repo.Name(), "/", 2)
+	allowSchema2 := true
```

Comment:
blast from the past. We need version 2 schema 2 support. What is this going to break at this point?